pushjack/apns.py

```diff
@@ -437,49 +437,62 @@ def sendall(self, stream, error_timeout=10):
 
 
 class APNSMessage(object):
-    """APNS message object that serializes to JSON."""
+    """
+    APNs message object that serializes to JSON.
+    Reordered and message replaced with body.
+    """
     def __init__(self,
-                 message=None,
-                 badge=None,
-                 sound=None,
-                 category=None,
-                 content_available=None,
```

Comment:
Not so keen on changing the ordering of these arguments. I'd prefer just adding the new ones at the end and keeping the existing ones where they are unless there is a compelling reason to? The downside is that any reordering can result in potential backwards compatibility issues for users (which I want to avoid unless there's a very good reason to make the change).